tests/unit/lib/test_datachain.py

```diff
@@ -1228,3 +1228,60 @@ class Nested(BaseModel):
             traces_double=[[{"x": 0.5, "y": 0.5}], [{"x": 0.5, "y": 0.5}]],
         )
     ]
+
+
+def test_rename_non_object_column_name_with_mutate(catalog):
+    ds = DataChain.from_values(ids=[1, 2, 3])
+    ds = ds.mutate(my_ids=Column("ids"))
+
+    assert ds.signals_schema.values == {"my_ids": int}
+
+    ds.save("mutated")
+
+    ds = DataChain(name="mutated")
+    assert ds.signals_schema.values == {"my_ids": int, "sys": Sys}
```

Comment:
please avoid touching parts that do not related to the test (sys). It can be just:
```python
assert ds.signals_schema.values.get("my_ids", None) == int
```

Reply:
Done